wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html

```diff
@@ -11,7 +11,7 @@
 
     <div class="privacy-indicator {% if is_public %}public{% else %}private{% endif %}">
         {% trans "Privacy" %}
-        <button type="button" data-a11y-dialog-show="set-privacy" class="status-tag primary">
+        <button type="button" data-a11y-dialog-show="set-privacy" class="button button-small button-secondary primary">
```

Comment:
```suggestion
        <button type="button" data-a11y-dialog-show="set-privacy" class="button button-small button-secondary">
```

I think the `primary` should also be removed

Reply:
ok sure

Reply:
Hey, I remove it and updated the branch.